backend/python/app/tests/graphql/mutations/test_story_mutation.py

```diff
@@ -0,0 +1,164 @@
+import re
+
+from ....models.story import Story
+
+
+def test_create_story(app, db, client):
```

Comment:
Lots of concerns here:

- Well this test fails for a type mismatch. Is this expected?
```
At index 0 diff: {'id': 1, 'story_id': '1', 'line_index': 0, 'cont
ent': 'line 1'} != {'id': 1, 'story_id': 1, 'line_index': 0, 'content': 'line 1'
```
- This is feels kinda hacky, particularly how I'm trying to work around the snake case conversion, and nesting of the contents array. 
- What do we actually want to test here? I've basically asserted that the db entry and response against expected. Is that right?